app/src/main/java/org/oppia/app/story/StoryActivity.kt

```diff
@@ -15,11 +15,11 @@ class StoryActivity : InjectableAppCompatActivity(), RouteToExplorationListener
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    val internalProfileId = intent.getIntExtra(STORY_ACTIVITY_INTENT_EXTRA_INTERNAL_PROFILE_ID, -1)
-    val topicId = checkNotNull(intent.getStringExtra(STORY_ACTIVITY_INTENT_EXTRA_TOPIC_ID)) {
+    val internalProfileId = intent.getIntExtra(STORY_ACTIVITY_INTENT_EXTRA_INTERNAL_PROFILE_ID_KEY, -1)
```

Comment:
Please take a look at this issue description more closely. There it is suggested to rename like this:
`STORY_ACTIVITY_INTENT_EXTRA_INTERNAL_PROFILE_ID` 
to
`STORY_ACTIVITY_INTERNAL_PROFILE_ID_KEY `. 

`EXTRA` word is also removed.